source/stylesheets/refills/_accordion-tabs-minimal.scss

```diff
@@ -1,14 +1,11 @@
 .accordion-tabs-minimal {
-  ///////////////////////////////////////////////////////////////////////////////////
   $base-border-color: gainsboro !default;
   $base-background-color: white !default;
   $base-spacing: 1.5em !default;
-  $base-accent-color: #477DCA !default;
-  $base-link-color: $base-accent-color !default;
+  $action-color: #477DCA !default;
```

Comment:
Any reason Hound’s comments were acted on before merging in? These should definitely be lowercase.